app/build.gradle

```diff
@@ -275,6 +275,14 @@ android.applicationVariants.all { variant ->
     addSecretToBuildConfig(variant, 'POCKET_KEY', ".pocket_key")
 }
 
+// -------------------------------------------------------------------------------------------------
+// Gecko: Put GeckoVersion into BuildConfig
+// -------------------------------------------------------------------------------------------------
+
+android.applicationVariants.all {
```

Comment:
Could we filter this to Gecko variants only?

Reply:
@Baron-Severin then we would have conditional BuildConfig :/. all checks should be split into variants